packages/renderer/src/lib/image/BuildImageFromContainerfileCards.svelte

```diff
@@ -133,7 +133,7 @@ function addCard(item: { value: string }): void {
 
 <div class="flex flex-col" role="region" aria-label="Build Platform Options">
   <div class="flex flex-row gap-x-4 gap-y-4 flex-wrap">
-    {#each sortedCards as card}
+    {#each sortedCards as card, index (`${card.title}-${index}`)}
```

Comment:
In this scenario, I would say, we should probably add a unique `id` to the card interface, and makes it unique (and ensure it is unique through a test)